addons/l10n_pl/models/account_move_line.py

```diff
@@ -0,0 +1,23 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, fields
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    def _compute_currency_rate(self):
```

Comment:
I think we need a `@api.depends('move_id.delivery_date')` here, since `delivery_date` is not in the dependencies of this compute method in the original definition on `account.move`

Reply:
Done and added a test like in CZ Pr